Capnp.Net.Runtime/PackedStream.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System.IO;
+
+namespace Capnp
+{
+    public static class PackedStream
```

Comment:
The name `PackedStream` might be misleading to users: It appears like a `Stream`-derived class which one can instantiate. In reality it is just a static utility class. Suggestion: Adding a plural "s" gives `PackedStreams` which might be less misleading. Or `PackedStreamUtil`?